plugins/steve/resource-instance.js

```diff
@@ -604,6 +608,18 @@ export default {
     };
   },
 
+  isSystemResource() {
+    if ( SYSTEM_NAMESPACES.includes(this.metadata?.namespace) ) {
+      return true;
+    }
+
+    if ( this.metadata?.namespace?.endsWith('-system') ) {
```

Comment:
If the user creates any namespace ending in -system, we will treat it as a system namespace - I don't think we want to do that.

Reply:
Now, I'm using v1/management.cattle.io.settings/system-namespaces to determine if it's a system namespace。

Reply:
Fixed